api/src/interfaces/cardRating.interface.ts

```diff
@@ -0,0 +1,8 @@
+import mongoose from "mongoose";
+
+export interface CardRating {
+    //Reference to pokemon card
+    pokemonCardId: mongoose.Types.ObjectId | string;
+    date: Date; 
+    rating: number; 
+  }
```

Comment:
Good job going above and beyond. Probably wasn't needed to make a CardRating Schema but good practice regardless.